spyder/plugins/editor/widgets/editorstack/editorstack.py

```diff
@@ -778,7 +830,161 @@ def display_help(self, help_text, clicked):
 
         self.send_to_help(name, help_text, force=True)
 
-    #  ------ Editor Widget Settings
+    # ---- Editor Widget Settings
+    @on_conf_change(
+        option=[
+            'highlight_current_line',
+            'highlight_current_cell',
+            'occurrence_highlighting',
+            'occurrence_highlighting/timeout'
+        ]
+    )
```

Comment:
Actually, since the methods are defined here I think I could just decorate them directly (no need for intermediate methods at all). What do you think about that @ccordoba12 ?